docs/reference/release-notes/6.8.asciidoc

```diff
@@ -34,6 +34,11 @@ This issue is fixed in Elasticsearch 6.8.10 and 7.7.1.
 
 Also see <<breaking-changes-6.8,Breaking changes in 6.8>>.
 
+* Java based - formats with '8' prefix - week based parsing and calculations are using JDK default calendar data provider which is Sunday,1.
+Sunday is considered first day of a week and it requires only 1 day in a week to for the first week of the year.
+It can be worked around by using locale which is based on ISO8601 rule (Monday,4) - for instance en-GB
+This issue is fixed in Elasticsearch 7.7 https://github.com/elastic/elasticsearch/pull/48209
```

Comment:
fair point, I will try to rephrase this in another doc PR.
Previously the parsing would fail with exception.
This PR fixes the bug so that parsing does not fail, but is uses Sunday,1 rule. So results are slightly different around when the year or week starts.

The 'ultimate' fixes are in 7.7 when IsoCalendarDataProvider is used and `-Djava.locale.providers=SPI,COMPAT` is set in jvm options